cylc/flow/task_job_mgr.py

```diff
@@ -1002,14 +1002,7 @@ def _simulation_submit_task_jobs(self, itasks, workflow):
             itask.submit_num += 1
             self._set_retry_timers(itask)
 
-            # We largely want a clean "platform" and not localhost,
-            # but we need to get the existing submission retry delays
-            # to stop _set_retry_delays missing it on retry.
-            itask.platform = {
-                'name': 'SIMULATION',
-                'submission retry delays':
-                    itask.platform['submission retry delays']
-            }
+            itask.platform = {'name': 'SIMULATION'}
```

Comment:
I thought this was needed for submission retry delays?

Reply:
https://github.com/cylc/cylc-flow/pull/5712#discussion_r1360622309

Reply:
You added `submission retry delays` here in an earlier commit, then removed them later.

Reply:
See this: https://github.com/cylc/cylc-flow/pull/5712#discussion_r1322582113

Reply:
Are submission retry delays supposed to impact simulation mode?

No

    Can simulation tasks submit-fail?

No

Nor can I recreate the scenario where there was a problem. Have removed this.